### PR TITLE
Add TIME (BEP20)

### PR DESCRIPTION
**Pull request by LeoTian888**
Adding info for BEP20 token 'TIME'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by LeoTian888** [comment]
https://github.com/trustwallet/assets/pull/19630
Please see this PR, the fee is already paid.
And the Certik audit is
[TimeAudit.pdf](https://github.com/trustwallet/assets/files/8994034/TimeAudit.pdf)

**Comment by Cryptocool1** [comment]
correction needed:
at least 2 links are required, 1 present. Add as many as you can: twitter, github, telegram, reddit, etc

File: blockchains/smartchain/assets/0x26619FA1d4c957C58096bBbeCa6588dCFB12E109/info.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "name": "TIME",
+    "type": "BEP20",
+    "symbol": "TIME",
+    "decimals": 18,
+    "website": "https://meta-time.org/",
+    "description": "Time token is issued by Meta Time (Cayman)",
```
Comment (cryptomanz) [inline blockchains/smartchain/assets/0x26619FA1d4c957C58096bBbeCa6588dCFB12E109/info.json:7]: description should be about your token

Comment (LeoTian888) [reply blockchains/smartchain/assets/0x26619FA1d4c957C58096bBbeCa6588dCFB12E109/info.json]: description updated, please have a look, thanks

**Pull request merged**